manifest.json

```diff
@@ -16,10 +16,23 @@
                 "http://*/*",
                 "https://*/*"
             ],
+            "include_globs": [
+                "*://localhost/*",
+                "*://*.local/*",
+                "*://10.*.*.*/*",
+                "*://172.16.*.*/*",
+                "*://172.17.*.*/*",
+                "*://172.18.*.*/*",
+                "*://172.19.*.*/*",
+                "*://172.2?.*.*/*",
+                "*://172.30.*.*/*",
```

Comment:
- `[]()![brand-devcloud-what-can-you-do-16x9.png.rendition.intel.web.864.486.png](https://user-images.githubusercontent.com/71736608/104387269-09bee600-5537-11eb-8eaf-2ecdf1238e42.png)`